modules/openapi-generator/src/main/resources/typescript-node/models.mustache

```diff
@@ -172,7 +172,9 @@ export class ObjectSerializer {
             let attributeTypes = typeMap[type].getAttributeTypeMap();
             for (let index in attributeTypes) {
                 let attributeType = attributeTypes[index];
-                instance[attributeType.name] = ObjectSerializer.deserialize(data[attributeType.baseName], attributeType.type);
+                if (attributeType.name) {
```

Comment:
perhaps a more robust fix could be to switch from `for .. in` to `for` here.
since you get some random property from `attributeTypes`, it's totally possible that this property also has a nested `name` key - so this check would be true, which is undesired.

Reply:
attributeTypes is just an array of the key's name and the value's type. I modified my fix for this to changing the for...in loop to a regular for loop, as that does not pick up the "ghost" property on the array like the for...in loop.  This also negates the need for a truth check.

```
attributeTypes is  [
  { name: 'group', baseName: 'group', type: 'string' },
  { name: 'name', baseName: 'name', type: 'string' },
  { name: 'description', baseName: 'description', type: 'string' }
]
```